Source/AlertControllerView.swift

```diff
@@ -77,11 +77,16 @@ class AlertControllerView: UIView {
     }
 
     private func createBackground() {
+      if let color = self.visualStyle.backgroundColor {
+        self.backgroundColor = color
+      }
+      else {
```

Comment:
Please follow this notation here:

`if let ... {
    // do whatever
} else {
    // do something else
}
```

Also please use 4 spaces to indent.